tests/codegen/builtin_func_wild.cpp

```diff
@@ -5,6 +5,7 @@ namespace test {
 namespace codegen {
 
 using ::testing::Return;
+using ::testing::_;
```

Comment:
This isn't needed

Reply:
Ok, I messed something up in my commits which re-added these. I'll remove and check if I missed anything else, I'll let you know when it's ready for review again.